apiserver/plane/app/views/workspace/preference.py

```
@@ -0,0 +1,49 @@
+# Module imports
+from ..base import BaseAPIView
+from plane.db.models.workspace import WorkspaceHomePreference
+from plane.app.permissions import allow_permission, ROLE
+from plane.db.models.workspace import WorkspaceHomePreference
```

Ruff (0.8.2)

Redefinition of unused `WorkspaceHomePreference` from line 3

Remove definition: `WorkspaceHomePreference`

(F811)